cocoa/NSPasteboard.go

```diff
@@ -3,11 +3,12 @@ package cocoa
 import (
 	"github.com/progrium/macdriver/core"
 	"github.com/progrium/macdriver/objc"
+	"log"
 )
 
 // The supported pasteboard types.
 // https://developer.apple.com/documentation/appkit/nspasteboardtype?language=objc
-type NSPasteboardType = string
+type NSPasteboardType string
 
 /*
 Code to get the values for those constants:
```

Comment:
lets move this example/how-to code to the wiki, and reference it here. also i can change this to a go program once on the wiki.

Reply:
As mentioned in Discord i think i need permission to edit/add wiki pages.

Reply:
ah, i forgot that was an option now. should be open. also adding you as collaborator